.circleci/config.yml

```diff
@@ -573,7 +573,7 @@ jobs:
       - checkout
       - install-prereqs
       - create-kind-clusters:
-          version: "v1.24.4"
+          version: "v1.25.3"
```

Comment:
latest release is [1.25.4](https://kubernetes.io/releases/), just want to verify if that should be reflected here?

Reply:
Oh weird, I looked a couple hours ago and `1.25.3` was the latest! let me look..

Reply:
From the kind release page:
`The default node image is a Kubernetes v1.25.3`  .. so I guess it makes sense to stay with the defaults for the latest version of Kind?